### ISSUE TITLE
is dirname really needed ?

### ISSUE THREAD

**Issue by vtorri** [luser-dr00g/xpost#52]
in configure.ac, i can see:

```
if ! test "x${ac_cv_func_dirname}" = "xyes" ; then
   AC_MSG_ERROR([dirname() function is mandatory, exiting...])
fi
```

but dirname() is not used at all (git grep dirname)

should we remove that check ?

**Comment by luser-dr00g** [comment]
Yes, it looks like it's not used anymore so it isn't needed.
I think it got removed when we switched to an environment variable to find
the /data directory instead of trying to find the path to the executable
and finding the relative path from there.
It can be removed.

**Comment by vtorri** [comment]
and I guess that libgen.h check can also be removed. I think that it was for dirname()

**Comment by luser-dr00g** [comment]
Yes, it looks like the only things available in libgen.h are basename() and dirname(). Oracle appears to have a special version with lots of crazy functions and the Posix version may have once held regcmp() and regex(). But yeah, we're not using any of that stuff.

Just using the ENV made everything so much easier. :)

**Comment by vtorri** [comment]
fixed in git
